Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function( grunt ) {
+
+grunt.initConfig( {
+	pkg: grunt.file.readJSON( "package.json" ),
+	jshint: {
+		files: [ "*.js", "routes/**.js" ],
+		options: {
+			jshintrc: ".jshintrc"
+		}
+	},
+	jscs: {
+		src: [ "*.js", "routes/**.js", "static/scripts/**.js" ]
+	},
+	csslint: {
+		files: [ "static/stylesheets/**.css" ],
+		options: {
+			csslintrc: ".csslintrc"
+		}
+	},
+	htmllint: {
+		src: [ "static/**/*.html" ]
+	}
+
+} );
+grunt.loadNpmTasks( "grunt-contrib-jshint" );
+grunt.loadNpmTasks( "grunt-jscs" );
+grunt.loadNpmTasks( "grunt-html" );
+grunt.loadNpmTasks( "grunt-contrib-csslint" );
```

Comment:
Do we really need that? The grunt config is super tiny and won't get super big anyways.